install.sh

```
@@ -1,96 +1,143 @@
-#!/bin/bash
+#!/usr/bin/env bash
+set -euo pipefail
 
-VERSION="0.1.10"
-set -e
+REPO="Enkryptify/cli"
+BIN_NAME="ek"
```

Shellcheck (0.11.0)

[warning] BIN_NAME appears unused. Verify use (or export if used externally).

(SC2034)